plugins/sigma.renderers.linkurious/canvas/sigma.canvas.nodes.def.js

```diff
@@ -194,6 +206,10 @@
       context.arc(x, y, size, 0, Math.PI * 2, true);
       context.closePath();
       context.fill();
+
+      if (borderSize > 0) {
```

Comment:
ok so the test would be `if (!node.active && borderSize > 0) {` so that the border is not drawn twice if the node is *active*.

FYI see [sigma.plugins.activeState](https://github.com/Linkurious/linkurious.js/tree/develop/plugins/sigma.plugins.activeState) and [sigma.plugins.select](https://github.com/Linkurious/linkurious.js/tree/develop/plugins/sigma.plugins.select)